roles/istio/tasks/certificates.yaml

```diff
@@ -1,4 +1,5 @@
 ---
+# Can't pass in external_ip like other roles as it comes from the Istio role itself, needs to be fetched here.
```

Comment:
:thought_balloon: The certs (and self-signed certs) tasks are at the end of the Istio tasks, meaning the functionality is additional not essential.
So, we _could_ make this work by either splitting the role into two, or by splitting the task list into two - those are functionally the same, the difference is in how they're called.

Given that there's a var controlling `istio_self_signed_certs`, this wouldn't be hard to change.

Reply:
Yeah would you prefer me to separate this into another role?

Reply:
Let's just make sure we have a ticket to follow on this refactor.

Reply:
Apologies @majolo, it didn't refresh me the page to show your comment in between my comments. 

I guess it would make sense to pull all certificates logic outside of Istio role.

Reply:
Hey so just picking up where this left off - I wanted to create a certificates role which managed certmanager + these self-signed certs but right now the certmanager work is a bit up in the air depending on what solution I come up with for the rate limits. Therefore I'd rather create a ticket to do this later once I know what's happening with certmaanager and get this PR merged today. What do you think?

Reply:
https://seldonio.atlassian.net/browse/DPL-981